common/config.h

```diff
@@ -60,7 +61,7 @@
 #define TDNF_REPO_KEY_RETRIES             "retries"
 #define TDNF_REPO_KEY_MINRATE             "minrate"
 #define TDNF_REPO_KEY_THROTTLE            "throttle"
-#define TDNF_REPO_KEY_SSL_VERIFY          "sslverify"
+#define TDNF_REPO_KEY_SSL_VERIFY          TDNF_CONF_KEY_SSL_VERIFY
```

Comment:
please keep this next to TDNF_CONF_KEY_SSL_VERIFY

Reply:
No, this is a `TDNF_REPO_KEY_` macro, why should this be under `TDNF_CONF_KEY_` macros?

Reply:
because they are same, will be easy to read if they are together

Reply:
I think it would be confusing. I want main options together, and repo options together.

Maybe we could add a section for common options, in another change when we add more.